### PR TITLE
Mention extra deprecated tags on profiling migration.

### PR DESCRIPTION
**Pull request by AlexJF**
### What does this PR do?
Mentions extra API key configuration environment variables and flags that need to be removed to properly migrate profiling from agentless setup to agent-based setup.

### Motivation
Our docs guiding users from agentless to agent-based profiling were missing some entries for some api key configuration alias fields. This was confusing users as they were continuing to see profiles being uploaded in the deprecated fashion after apparently having followed all the instructions.

### Preview

https://docs-staging.datadoghq.com/alexjf/prof-2641/tracing/faq/profiling_migration/?tab=java

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by kayayarai — APPROVED** [review]
Docs look fine. Will wait for @GaurabAryal to also approve before merging. One question: Do similar change need to be made on the Python and Go tabs as well, or does this just apply to Java?

**Comment by AlexJF** [comment]
The Python and Go instructions already mentioned `DD_API_KEY` (and IIRC, they never supported the file-based version to start with) so they should be good! :+1:

**Comment by GaurabAryal** [comment]
Thanks @AlexJF for fixing :). @kayayarai good to go on my end!

**Review by GaurabAryal — APPROVED** [review]

**Pull request merged**
